### PR TITLE
Bug 1142558 - Intermittent failing test, TEST-UNEXPECTED-FAIL | /builds/...

### PR DESCRIPTION
**Pull request by adrm**
...slave/test/gaia/apps/communications/contacts/test/marionette/ice_test.js | Contacts > ICE contacts ICE settings Check ICE settings transition

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=98ed497469c3)

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=aa70ee11ac88)

File: apps/communications/contacts/test/marionette/ice_test.js [hunk 1]
```diff
@@ -58,17 +58,18 @@ marionette('Contacts > ICE contacts', function() {
 
   function addTestContact(addTelephone) {
     var tel = addTelephone ? testContactTelNumber : undefined;
-    subject.addContact({
-      givenName: 'Hello',
-      tel: tel
+    createMozContact({
+      givenName: ['Hello'],
+      tel: [{value: tel}]
```
Comment (jmcanterafonseca) [inline apps/communications/contacts/test/marionette/ice_test.js]: better if we provide a type here as well

File: apps/communications/contacts/test/marionette/ice_test.js [hunk 2]
```diff
@@ -383,23 +381,30 @@ marionette('Contacts > ICE contacts', function() {
...
       assertICEContact(position, 'Hello');
     }
 
     function editAndMatchTest(position) {
-      subject.addContact({
-        givenName: 'Good Bye',
-        tel: '+34638883074'
+      var contactID = createMozContact({
+        givenName: ['Good Bye'],
+        tel: [{value:'+34638883074'}]
```
Comment (jmcanterafonseca) [inline apps/communications/contacts/test/marionette/ice_test.js]: we would need a type here as well

**Comment by try-server-hook** [comment]
[ADLR-es Adrián de la Rosa (ADLR-es)](https://github.com/ADLR-es) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a5990a31440a)

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b04dda886b87664ae9f16fce9a0414a7ed5dc80f

**Pull request merged**